.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+* [ ] I have read the [guidelines].
+
+<!--
+    We categorize our issues into 2 categories:
+    * Bug: Prefix your issue title with [BUG]
+    * Feature Request: Prefix your issue title with [FEATURE]
+    * Question: Prefix your issue title with [QUESTION]
```

Comment:
No, please don't add questions. Issues are issues - GitHub issues is not a forum.